x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_flyout.ts

```diff
@@ -32,6 +32,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const excludeKeysFromServerless = ['size']; // https://github.com/elastic/kibana/issues/178954
 
   describe('Dataset quality flyout', () => {
+    this.tags(['failsOnMKI']); // Failing https://github.com/elastic/kibana/issues/183495
```

Comment:
This will probably fail as it doesn't work with arrow functions. Should rather be

```suggestion
  describe('Dataset quality flyout', function () {
    this.tags(['failsOnMKI']); // Failing https://github.com/elastic/kibana/issues/183495
```